### PR TITLE
haha

### PR DESCRIPTION
**Pull request by owner6**

### CODE REVIEW

File: scrypt.js [hunk 1]
```diff
@@ -2,4 +2,5 @@ document.body.innerHTML = '<h1>hello js</h1>';
 console.log('test1')
 console.log('test2!!!!!!!!!!')
 console.log('test2!!!!!!!!!!')
-console.log('test3')
\ No newline at end of file
+console.log('test3')
+console.log('test45')
```
Comment (dever4eg) [inline scrypt.js]: It is better to add an empty line to the end of the file. You can see that github also suggests it
image

File: scrypt.js [hunk 2]
```diff
@@ -2,4 +2,5 @@ document.body.innerHTML = '<h1>hello js</h1>';
 console.log('test1')
 console.log('test2!!!!!!!!!!')
 console.log('test2!!!!!!!!!!')
```
Comment (Dever4eg) [inline scrypt.js:4]: Maybe let's remove duplicated console.log?

Comment (dever4eg) [inline scrypt.js]: Maybe let's remove duplicated console.log?

**Review by dever4eg — CHANGES_REQUESTED** [review]

File: scrypt.js (hunk 2 above)
Comment (dever4eg) [reply scrypt.js]: @owner6 what do you think? Do we need the same console log twice?

Comment (owner6) [reply scrypt.js]: I think let's delete it

**Review by dever4eg — APPROVED** [review]

**Pull request merged**